src/migrations/20190705125652-create-pa-table.ts

```diff
@@ -0,0 +1,42 @@
+import { DataTypes, NOW, QueryInterface } from "sequelize";
+
+export default {
+  up: (queryInterface: QueryInterface): Promise<void> => {
+    return queryInterface.createTable("PA", {
+      createdAt: {
+        allowNull: false,
+        type: new DataTypes.DATE()
+      },
+      deletedAt: new DataTypes.DATE(),
+      description: {
+        allowNull: false,
+        type: new DataTypes.STRING()
+      },
+      ipaCode: {
+        allowNull: false,
+        field: "ipa_code",
+        primaryKey: true,
+        type: new DataTypes.STRING()
+      },
+      name: {
+        allowNull: false,
+        type: new DataTypes.STRING()
+      },
+      pec: {
+        allowNull: false,
+        field: "PEC",
+        type: new DataTypes.STRING()
+      },
+      registrationDate: {
```

Comment:
for consistency with other date fields:

```suggestion
      registeredAt: {
```

Reply:
what's the difference between `createdAt` and `registeredAt` ?